brigade-api/cmd/brigade-api/main.go

```diff
@@ -20,6 +20,14 @@ import (
 )
 
 var (
+	// DefaultCORS defines the default CORS settings for the API
+	DefaultCORS = restful.CrossOriginResourceSharing{
+		AllowedHeaders: []string{"Content-Type", "Accept"},
```

Comment:
@bacongobbler - does this allow requests from all origins by default?

Reply:
Yep! I'm not sure if that's a good default, but that's the default setting from go-restful.